resources/views/auth/admission/index.blade.php

```diff
@@ -54,13 +50,13 @@
         </div>
     @endif
     @foreach($applications as $application)
-        <a href="{{route('applications', ['id' => $application->user_id])}}">
+        <a href="{{route('admission.applicants.show', ['application' => $application->id])}}">
             @include('auth.application.application', ['user' => $application->user, 'expanded' => false])
         </a>
     @endforeach
     <hr>
     <h6>Összesen: <b class="right">{{$applications->count()}} jelentkező</b></h6>
-    @can('finalize', \App\Models\ApplicationForm::class)
+    @can('finalize', \App\Models\Application::class)
         @if($applicationDeadline?->addWeeks(1) < now())
             <div class="card" style="margin-top:20px">
                 <div class="card-content">
```

Comment:
Here, the comments about the "Submitted" and "Called in" statuses are not up-to-date.

Reply:
See https://github.com/EotvosCollegium/mars/pull/574

Reply:
No, actually, you are right, I see what you mean now. I will fix this quickly.